README.md

```diff
@@ -1540,23 +1530,24 @@ JSON encoded body using the "application/json" content type.
     <td>optional</td>
   </tr>
   <tr>
-    <td>snapshot</td>
-    <td>element</td>
-    <td>Picture of the viewpoint</td>
+    <td>bitmaps</td>
```

Comment:
Actually, to get in line with the BCF XML spec, there should be multiple bitmaps allowed per viewpoint. (The snapshot is singular)